core/arch/arm/plat-imx/crypto_conf.mk

```diff
@@ -37,10 +37,12 @@ CFG_CAAM_SGT_ALIGN ?= 1
 CFG_NXP_CAAM_SGT_V1 ?= y
 
 ifeq ($(filter y, $(CFG_MX8QM) $(CFG_MX8QX)),y)
+$(call force, _CFG_CAAM_SIZE_ALIGN,4)
```

Comment:
Hi @jforissier,
By adding the underscore, I thought I followed the recommendation you made in a previous comment : https://github.com/OP-TEE/optee_os/pull/4871#discussion_r712105147 but apparently I misunderstood something. Could you clarify this rule for me please?

Reply:
Hmmm okay this is debatable indeed. I agree this variable could very well have an underscore since it isn't freely configurable at build time (it only depends on other variables). But isn't it also the case of `CFG_JR_BLOCK_SIZE` etc.? Introducing some inconsistency is not very good either.

Reply:
I see. Since I've only been aware on this rule recently, if I start applying this rule from now for i.MX platforms, we are going to have a lot of inconsistency indeed ...
To be honest, I don't see myself changing everything in i.MX to comply with this rule for now. Do you mind if we skip this rule and go back to what you originally suggested ?  https://github.com/OP-TEE/optee_os/pull/5113#discussion_r785766947

Reply:
I'm fine with that. To be honest this is not a strong rule, it initially started with the crypto stuff, lots of combinations especially with support for multiple libraries (LibTomCrypt, MBed TLS) and using internal variables `_CFG_*` helped clarify or rather avoid to complexify, but here it would not help.

Reply:
Ok, thanks! I try to keep track of every coding rule I could have encountered while upstreaming i.MX support. I keep this  rule in mind and I will apply it when all conditions are met.
In the meanwhile, I will remove the underscore prefix.